core-plugins/src/test/java/io/cdap/plugin/batch/source/FileBatchSourceTest.java

```diff
@@ -91,6 +96,20 @@
                                                               Schema.Field.of("l", Schema.of(Schema.Type.LONG)),
                                                               Schema.Field.of("file",
                                                                               Schema.of(Schema.Type.STRING)));
+
+  private static final Schema RECORD_SCHEMA_VALID_DATE =
+    Schema.recordOf("record",
+                    Schema.Field.of("i", Schema.of(Schema.Type.INT)),
+                    Schema.Field.of("timestamp", Schema.of(Schema.LogicalType.TIMESTAMP_MICROS)),
```

Comment:
Can you please verify that timestamp and datetime types in the input schema work correctly?